package.json

```diff
@@ -41,9 +44,11 @@
     "ember-cli-notifications": "^3.2.0",
     "ember-lodash": "~0.0.6",
     "ember-truth-helpers": "^1.2.0",
-    "eslint": "^2.1.0",
+    "eslint": "2.2.0",
     "eslint-config-frost-standard": "^1.0.0",
-    "ember-frost-css-core": "^1.0.0"
+    "frost-checkbox-array": "~1.1.0",
```

Comment:
Is checkbox array just used in the demo or in the addon itself? If it's just the demo could you replace it with any other sample cotent? Checkbox array is not on github (not open source) so we should avoid using it.